bazel/BUILD

```diff
@@ -699,6 +699,15 @@ selects.config_setting_group(
     ],
 )
 
+selects.config_setting_group(
```

Comment:
Had to introduce  a new setting group that includes various darwin architectures, just building with `"//bazel:darwin": [":maxmind"],` (already existing [darwin](https://github.com/envoyproxy/envoy/blob/main/bazel/BUILD#L580) property) did not include maxmind extension in the result binary on my M3/arm laptop.